schemas/context/profile-lead-crm.schema.json

```diff
@@ -0,0 +1,172 @@
+        },
+        "xdm:emailBouncedReason": {
+          "title": "Email Bounced Reason",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the reason for bounced email.",
+          "type": "string"
+        },
+        "xdm:jigsawContactID": {
+          "title": "Jigsaw Contact Id",
+          "description": "Lead’s Data.com ID if available.",
+          "type": "string"
+        },
+        "xdm:lastTransferDate": {
+          "title": "Last Transfer Date",
+          "description": "Last transfer date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:ownerID": {
+          "title": "Owner ID",
+          "description": "Owner id.",
+          "type": "string"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:leadStatus": {
+          "title": "Lead Status",
+          "description": "Field recording the current marketing/sales status of the lead.",
+          "type": "string"
+        },
+        "xdm:leadPhotoUrl": {
+          "title": "Lead Photo Url",
+          "description": "Lead Photo url.",
+          "type": "string"
+        },
+        "xdm:unReadByOwner": {
+          "title": "Read By Owner Indicator",
+          "description": "Read by owner indicator.",
+          "type": "boolean"
+        },
+        "xdm:ProspectAddress": {
+          "title": "Prospect Address",
+          "description": "Prospect address.",
+          "$ref": "https://ns.adobe.com/xdm/common/address"
+        },
+        "xdm:prospectCompany": {
+          "title": "Prospect Company",
+          "description": "Prospect company.",
+          "type": "string"
+        },
+        "xdm:prospectName": {
+          "title": "Prospect Name",
+          "description": "Prospect name.",
+          "$ref": "https://ns.adobe.com/xdm/context/person-name"
+        },
+        "xdm:prospectScore": {
+          "title": "Prospect Score",
+          "description": "Prospect score.",
+          "type": "number"
+        },
+        "xdm:prospectSource": {
+          "title": "Prospect Source",
+          "description": "Prospect source.",
+          "type": "string"
+        },
+        "xdm:prospectWebsite": {
+          "title": "Prospect Website",
+          "description": "Prospect website.",
+          "type": "string",
+          "format": "uri"
+        },
+        "xdm:annualRevenue": {
+          "title": "Annual Revenue",
+          "description": "Estimated Amount of annual revenue of the lead organization.",
+          "type": "number"
+        },
+        "xdm:leadOrganization": {
+          "title": "Organization Attributes",
+          "description": "Organization attributes if the lead is an organization.",
+          "$ref": "https://ns.adobe.com/xdm/common/organization"
+        }
```

Comment:
Additional OPT-OUT data type with reference to EMAIL, FAX and PHONE. Same referenced in the Contact PR.

Reply:
Instead of adding optinout in the allOf, I add a property "leadPreferences" in the CRM mixin and referred the optinout data type in that. I did the same with the contact as well to be consistent. We no longer need optinout in the allOf section now.